src/test/scala/org/neo4j/spark/SparkConnectorScalaBaseWithApocTSE.scala

```diff
@@ -58,7 +58,7 @@ class SparkConnectorScalaBaseWithApocTSE {
             val afterConnections = SparkConnectorScalaSuiteWithApocIT.getActiveConnections
             SparkConnectorScalaSuiteWithApocIT.connections == afterConnections
           }
-        }, Matchers.equalTo(true), 30, TimeUnit.SECONDS)
+        }, Matchers.equalTo(true), 45, TimeUnit.SECONDS)
```

Comment:
perché lo hai alzato?

Reply:
mi è andato in timeout un paio di volte con le APOC